src/Controls/src/Core/MultiPage.cs

```diff
@@ -85,7 +85,7 @@ public T CurrentPage
 				OnPropertyChanging();
 
 				// TODO: MAUI refine this to fire earlier
-				_current?.SendNavigatingFrom(new NavigatingFromEventArgs());
+				_current?.SendNavigatingFrom(new NavigatingFromEventArgs(_current, NavigationType.Replace));
```

Comment:
Same here. 

Not going to comment on the rest the `SendNavigatingFrom` ones need to all be fixed